### DISCUSSION TITLE
Create a special characters section

### DISCUSSION THREAD

**Discussion by lucaspessivaldo** [monkeytypegame/monkeytype#4184] [Ideas] [answered]
The weak point of most people, while typing, is the special characters. I propose to create a section **Special characters** or another name. That just focuses on special characters.
If yes, I can make the Pull request.

The special characters
! @ # $ % ^ & * ( ) - _ = + [ { ] } \ | ; : ' " , < . > / ? ` ~ 

**![Screenshot from 2023-04-15 11-18-58](https://user-images.githubusercontent.com/11798065/232230117-367623f7-dcae-43f3-bf09-7776070029d5.png)**

**Comment by Miodec** [comment, marked as answer]
Try the specials or ascii funboxes

**Comment by lucaspessivaldo** [comment]
That's cool, I didn't even know that Monkeytype had this.
I think **ascii** should be on the main languages. What do you think about it?

**Reply by Miodec** [reply]
No, with the way languages are setup in code, you would need to add a lot of combinations to the language file. Funboxes generate new "words" on the fly.
